windows.c

```diff
@@ -97,7 +101,7 @@ int initialize(INTERNET_PER_CONN_OPTION_LIST* options) {
 int query(INTERNET_PER_CONN_OPTION_LIST* options) {
   DWORD dwBufferSize = sizeof(INTERNET_PER_CONN_OPTION_LIST);
   if(!InternetQueryOption(NULL, INTERNET_OPTION_PER_CONNECTION_OPTION, options, &dwBufferSize)) {
-    reportWindowsError("Querying options");
+    reportWindowsError("querying options", options->pszConnection);
```

Comment:
No, the branch is still required in cases like [this](https://github.com/getlantern/sysproxy-cmd/pull/6/files#diff-23ba024365564263bc3bb4665d30db3dR179)